apis/goharbor.io/v1alpha2/harbor_types.go

```diff
@@ -486,6 +486,9 @@ type HarborExposeComponentSpec struct {
 	Ingress *HarborExposeIngressSpec `json:"ingress,omitempty"`
 
 	// TODO Add supports to ClusterIP, LoadBalancer and NodePort by deploying the nginx component
+
+	// +kubebuilder:validation:Optional
+	LoadBalancer *HarborExposeLBSpec `json:"loadbalancer,omitempty"`
```

Comment:
It seems we need to add some logic in the validation webhook to make sure the two types cannot be configured at the same time.